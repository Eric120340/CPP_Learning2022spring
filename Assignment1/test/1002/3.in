8 6
35821 18763 20546 58794 52184 48332 
19840 45964 85507 85237 26737 16378 
17317 80350 5597 95712 64100 8403 
73730 59033 52620 70634 89885 25324 
35822 80579 75824 94951 83748 29821 
37707 19568 48583 74604 94713 17119 
39287 30904 63082 24793 16141 6170 
41170 49809 2871 63119 45520 83323 
8 6
87873 35601 42355 40492 6234 32239 
82167 58407 12818 57990 53357 12917 
4163 91063 32484 69097 82018 43548 
86215 21305 74452 65648 46097 6944 
88170 3619 56752 91040 83089 18623 
90714 70961 54223 33069 27804 76808 
81659 9970 35214 10828 84311 88570 
23744 4825 95985 72579 73922 78002 
