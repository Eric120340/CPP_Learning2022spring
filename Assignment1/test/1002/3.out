123694 54364 62901 99286 58418 80571 
102007 104371 98325 143227 80094 29295 
21480 171413 38081 164809 146118 51951 
159945 80338 127072 136282 135982 32268 
123992 84198 132576 185991 166837 48444 
128421 90529 102806 107673 122517 93927 
120946 40874 98296 35621 100452 94740 
64914 54634 98856 135698 119442 161325 
